packages/datatrak-web/src/Routes.tsx

```diff
@@ -69,7 +69,7 @@ export const Routes = () => {
           <Route index element={<LandingPage />} />
         </Route>
         {/** Any views that should have the background image should go in here */}
-        <Route path="/" element={<BackgroundPageLayout />}>
+        <Route path="/" element={<AuthPageLayout />}>
           {/** Any public centred views should go in here */}
           <Route path="/" element={<CentredLayout />}>
```

Comment:
The centred layout is not laying out quite right on shorter screens such as laptop. The paper should be vertically centred with padding around it and there should be a scrollbar inside the paper.

Reply:
This still isn't quite right unfortunately. It has padding now but it still doesn't fit inside the screen (mine is 1200px high). I think that some where there needs to be a height: calc(100% - HEADER_HEIGHT) or something to make it fit.

Also it is very minor but in the design the Paper is flat with a border while in the app it has a box shadow (eg. Paper component needs variant=outlined prop)

<img width="1440" alt="image" src="https://github.com/beyondessential/tupaia/assets/12807916/9213d7eb-c2b4-440a-9fdb-8c2a4ca16ca0">

Reply:
Sorry, my bad, fix has been pushed